### DISCUSSION TITLE
トレンドの記事投稿日が間違っている

### DISCUSSION THREAD

**Discussion by hiroki0525** [increments/qiita-discussions#125] [要望・アイデア・不具合]
2022年1月31日に投稿した記事が[トレンドページ](https://qiita.com/trend)に表示されましたが、記事投稿日が**2022年1月30日**になっています。

<img width="1017" alt="スクリーンショット 2022-01-31 19 07 58" src="https://user-images.githubusercontent.com/40714517/151774679-9d777702-8e1c-4d34-9c5a-4e5e973f7f49.png">

[記事の詳細ページ](https://qiita.com/nuko-suke/items/ca42c0ad963ee3c19dbd)では記事投稿日が正しく**2022年1月31日**で表示されています。

<img width="750" alt="スクリーンショット 2022-01-31 19 09 40" src="https://user-images.githubusercontent.com/40714517/151774924-877b58eb-2231-45f4-b471-bf4400869202.png">

お手数ですがご確認お願いいたします。

※ブラウザのキャッシュを削除してもダメでした。
※サーバーから返却されているHTMLを見る感じは `2022-01-30T22:08:54Z` になっています。ちょうどその9時間後に記事を投稿しているので、恐らく東京のタイムゾーンへの変換漏れが気がします。

<img width="407" alt="スクリーンショット 2022-01-31 19 12 31" src="https://user-images.githubusercontent.com/40714517/151775524-d59ad516-1bed-44c2-95bd-530e397416cc.png">

**Comment by kskwtnk** [comment]
@hiroki0525 

ご意見と、調査までしていただきありがとうございます！

おそらく調べていただいたようにタイムゾーンの問題に思われますが、1度運営の方でもちゃんと確認して対応いたします！

**Comment by ohakutsu** [comment]
@hiroki0525

遅くなってしまい、申し訳ございません :bow:

こちらのバグは、トレンドなど記事カードを使っている部分で起きていることが確認できました。

バグの修正についてですが、ちょうど[記事カードのUI変更](https://blog.qiita.com/update-article-card-beta/) があったため、UI変更に合わせて修正を実施しました！
まだベータ版としての提供となっていますが、新しくなる記事カードでは正しく表示されるので今しばらくお待ちください！

**Comment by kato647** [comment]
@hiroki0525 
大変お待たせしました :bow:
ホーム・トレンドの記事の投稿日と記事ページの投稿日が異なる不具合を解消しました。
ホーム・トレンド画面をご確認いただければと思います！

**Reply by hiroki0525** [reply]
直っていそうですね！修正お疲れさまです！
